components/brave_wallet_ui/components/buy-send-swap/swap-input-component/index.tsx

```diff
@@ -474,35 +445,12 @@ function SwapInputComponent (props: Props) {
       }
       {componentType === 'toAddress' && toAddress !== toAddressOrUrl && !addressError &&
         <>
-          {showEnsOffchainLookupOptions
-            ? (
-              <>
-              <WarningRow>
-                  <WarningText>{getLocale('braveWalletEnsOffchainWarning')}</WarningText>
-              </WarningRow>
-              <span>
-                <EnsProceedButton onClick={onClickProceedEnsOffchainYes}>
-                  {getLocale('braveWalletEnsOffchainProceedWithYes')}
-                </EnsProceedButton>
-                <EnsProceedButton onClick={onClickProceedEnsOffchainNo}>
-                  {getLocale('braveWalletEnsOffchainProceedWithNo')}
-                </EnsProceedButton>
-                <LearnMoreButton onClick={onClickEnsOffchainLearnMore}>
-                  {getLocale('braveWalletAllowAddNetworkLearnMoreButton')}
-                </LearnMoreButton>
-              </span>
-              <Checkbox isChecked={ensOffchainOptionsRemember} onChange={setEnsOffchainOptionsRemember}>
-                {getLocale('braveWalletEnsOffchainDontShowAgain')}
-              </Checkbox>
-              </>
-            )
-            : (<Tooltip
+          <Tooltip
```

Comment:
I assumed we don't need to support this, so just removed ens offchain support at all